it/uk/gov/hmrc/breathingspaceifstub/support/BaseISpec.scala

```diff
@@ -41,7 +42,12 @@ trait BaseISpec
 
   implicit lazy val materializer: Materializer = app.materializer
 
-  lazy val localContext: String = "/breathing-space/api"
+  lazy val statefulLocalContext: String = "/breathing-space/api"
+  lazy val statelessLocalContext: String = "/breathing-space-stateless/api"
+
+  lazy val wsClient: WSClient = fakeApplication.injector.instanceOf[WSClient]
```

Comment:
I suppose you can simplify this to be just `... = inject[WSClient]`

Reply:
Done